R/hypothesize.R

```diff
@@ -0,0 +1,129 @@
+# hypothesize.R
```

Comment:
I don't understand what's happening here, without comments. There's a lot of to-and-fro between strings and factors and I'm not sure that's necessary...

Reply:
Hello, I simplified the function